SourceCode/Builds/build_with_msvc22.md

```diff
@@ -0,0 +1,90 @@
+# Building and Compiling C&C Generals & Zero Hour on Visual Studio 2022
+
+This guide will walk you through the process of basic setup and compilation of the C&C Generals and Zero Hour source
+code using Visual Studio 2022.
+For build using solutions and more advanced build configurations, see below.
+
+## Prerequisites
+
+1. **Visual Studio 2022**
+
+- Ensure that the necessary C++ development components, **including MFC**, are installed.
+
+>[!NOTE]
+> You must have the MFC components installed to compile the source code.
+<!-- markdownlint-disable-next-line -->
+2. **Obtain the Source Code**
+    - Clone or download the source code
+      repository: [TheSuperHackers - GeneralsGameCode](https://github.com/TheSuperHackers/GeneralsGameCode.git).
+
+## Steps
+
+### 1. Prepare the project
+
+- Open the cloned folder in Visual Studio 2022.
+- Wait for Visual Studio to generate the necessary CMake files.
+
+### 2. Build the Project
+
+- Select the appropriate build configuration:
+  - `Build Windows build` for a release build.
+  - `Build Windows Debug build` for a debug build.
+  - `Build Windows Internal build` for an internal build.
+  - `Build Windows Profile build` for a profile build.
+
+>[!TIP]
+> For more information on the different build configurations, see the [Build Configurations](build_configurations.md)
+page.
+
+- Select the target you want to build:
+  - `generalsv.exe` to build the base Generals.
+  - `generalszh.exe` to build Zero Hour.
+
+- Build the project by clicking on the `Build` menu and selecting `Build`.
+- The compiled executable will be placed in the `build/<win32 build type>/<game name (Generals/GeneralsMD)>/<build configuration>`
+  folder. Example: `build/win32dgb/GeneralsMD/Debug`
+- Install the game executable in the game directory by clicking on the `Install` in `Build` menu. This will copy the
+  executable to the retail game directory.
+
+## Additional Steps
+
+### Build through CMake target view
+
+- In the Solution Explorer, click on 'switch view' and select 'CMake Targets View'.
+- Expand the 'Genzh' project and right-click on the target you want to build.
+- Select 'Build' to compile the target.
+
+### Build using command line
+
+You need to install [CMake](https://cmake.org/download/) and [Ninja](https://github.com/ninja-build/ninja/releases)
+to build the project from the command line.
+
+- In the developer command prompt, open the settings to add the x86 environment terminal.
+
+>[!Tip]
```

Comment:
I would suggest moving the tip to below where the instructions say to open a new terminal from the terminal drop down as both alternatives end up at that point then? Otherwise the tip should say which point to skip to?